reference/v8.json

```diff
@@ -607,6 +608,11 @@
           "js": "0.10.0",
           "ios": "2.0.0",
           "android": "2.0.1"
+        },
+        "auto": {
+          "js": "TBD",
```

Comment:
Yeah that was my question in https://github.com/mapbox/mapbox-gl-style-spec/pull/493#issuecomment-247483816. Sounds like the process is to land this PR first and leave all 3 SDKs as `undefined`. Then, land the SDK PRs to support `auto`, then come back here and make a new PR to update support. Is my understanding correct?